docs/advanced_reference/commands/autonomy_service.md

```diff
@@ -38,8 +38,56 @@ This command group consists of a number of functionalities to manage the life cy
 
     The options `--use-ethereum`, `--use-goerli`, `--use-custom-chain` and `--use-local` are mutually exclusive.
 
-`--skip-hash-check`
-: Skip hash check when verifying dependencies on chain.
+`-t, --timeout FLOAT`
+: Timeout for verifying emitted events
+
+## `autonomy service info`
+
+Print service information.
+
+### Usage
+
+```bash
+autonomy service info [OPTIONS] SERVICE_ID
```

Comment:
what is "Options" here? I don't see any option. Missing how to choose the chain?